usmqe/gluster/gluster.py

```diff
@@ -202,7 +189,39 @@ def info(self):
 
         Returns:
             dictionary: parsed json from
-                        ``gluster --format json --cluster CLUSTERNAME status``
+                        ``gluster volume info VOLUMENAME --xml``
                         command
         """
-        return self.run_on_node('info')
+        xml = self.run_on_node('info {}'.format(self.name))
+        self.id = xml.findtext("./volInfo/volumes/volume/id")
+        LOGGER.debug("Volume_id: %s" % self.id)
+        self.status = xml.findtext(
+            "./volInfo/volumes/volume/statusStr")
+        self.stripe_count = xml.findtext(
+            "./volInfo/volumes/volume/stripeCount")
+        self.replica_count = xml.findtext(
+            "./volInfo/volumes/volume/stripeCount")
+        self.brick_count = xml.findtext(
+            "./volInfo/volumes/volume/brickCount")
+        self.snap_count = xml.findtext(
+            "./volInfo/volumes/volume/snapshotCount")
+        LOGGER.debug("Volume_status: %s" % self.status)
+
+    def get_volume_id(self):
```

Comment:
One note for these methods. They don's have to call info again, only if required. Several possible approaches:
1. Add some boolean attribute to the instance, the parameter will be instanced as 'False' in __init__ and as 'True' in info method or vice versa. The parameter says if the info method should be called or not.
2. Add some optional parameter to all of these methods
3. Combination of both previous

On the other hand I like what you've done with this module.